README.md

```diff
@@ -40,18 +38,30 @@ python main.py --input input.dxf --output output.png
 ```
 Both input and output files are set relative to the script's root directory, i.e. should be located in the same directory.
 
-#### Setup with Virtualenv
+#### Preparing the environment on Linux
+
+##### Setup with Virtualenv
 * Create a virtual environment: `virtualenv -p /usr/bin/python3 cad-mixer`
```

Comment:
For the environment name I would stick to `virtualenv -p /usr/bin/python3 venv` because how it is setup right now the virtualenv might get commited to the repository since it's not in the `.gitignore` file!